src/settings/MappingProfiles/MappingProfileDetailsRoute/MappingProfileDetailsRoute.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes-core';
+
+import { MappingProfileDetails } from '../MappingProfileDetails';
+
+const MappingProfileDetailsRoute = ({
+  onCancel,
+  resources: { mappingProfile },
+}) => {
+  return (
+    <MappingProfileDetails
+      onCancel={onCancel}
+      mappingProfile={mappingProfile || {}}
+    />
+  );
+};
+
+MappingProfileDetailsRoute.propTypes = {
+  onCancel: PropTypes.func.isRequired,
+  resources: PropTypes.object.isRequired,
+};
+
+MappingProfileDetailsRoute.manifest = Object.freeze({
+  initializedFilterConfig: { initialValue: false },
+  mappingProfile: {
+    type: 'okapi',
+    path: 'data-export/mappingProfiles/:{id}',
+    resourceShouldRefresh: true,
+    shouldRefresh: () => false,
```

Comment:
Could you please clarify what is this for exactly?

Reply:
I add this to string to resolve problem with render page previous details. I recheck it and see that it works normally without `shouldRefresh` and any difference in functionality, but `ui-data-import` use it.

Reply:
Let me try as well. Thanks.

Reply:
Looks good. Let's leave then.